var/spack/repos/builtin/packages/py-pyfftw/package.py

```diff
@@ -10,24 +10,31 @@ class PyPyfftw(PythonPackage):
     """A pythonic wrapper around FFTW, the FFT library,
     presenting a unified interface for all the supported transforms."""
 
-    homepage = "http://hgomersall.github.com/pyFFTW"
-    pypi = "pyFFTW/pyFFTW-0.10.4.tar.gz"
+    homepage = "https://pyfftw.readthedocs.io/en/latest/"
+    pypi = "pyFFTW/pyfftw-0.14.0.tar.gz"
```

Comment:
With the new `pypi` name, the older versions fail to download. This will need a `url_for_version` function, see e.g. `py-werkzeug` with
```py
    def url_for_version(self, version):
        url = "https://files.pythonhosted.org/packages/source/w/werkzeug/{0}-{1}.tar.gz"
        if version >= Version("2.3.7"):
            letter = "werkzeug"
        else:
            letter = "Werkzeug"
        return url.format(letter, version)
```

Reply:
I added a `url_for_version` method. Are you know able to confirm the checksums?